src/contexts/surveyStatus/index.js

```diff
@@ -0,0 +1,21 @@
+import React, { createContext, useReducer } from 'react';
+
+import SurveyStatusReducer from 'contexts/surveyStatus/reducer';
```

Comment:
How about moving the reducer to `/reducers` as mentioned on [Compass](https://nimblehq.co/compass/development/code-conventions/react/#project-structure) 🙌